src/DynamoPackagesUI/PackageManagerExtension.cs

```diff
@@ -55,7 +55,7 @@ private void AddMenuItem(ViewLoadedParams p)
 #endif
         }
 
-        private void OnPackageManagerClick()
+        public void OnPackageManagerClick()
```

Comment:
Just for testing, you don't need to make this method public. You can use InternalVisibleTo attribute.